app/src/main/java/com/example/android_demo_kotlin/Kotlin_Basic.kt

```diff
@@ -0,0 +1,258 @@
+package com.example.android_demo_kotlin
+
+import android.annotation.SuppressLint
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+
+class Kotlin_Basic : AppCompatActivity() {
+    @SuppressLint("LongLogTag")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_kotlin_basic)
+
```

Comment:
Remove extra space

Reply:
Check this same mistake in whole file and update it.

Reply:
Okk I Updated the File